doc/ecosystem.md

```diff
@@ -0,0 +1,131 @@
+# Ecosystem
+
+PyGMT provides a Python interface to the Generic Mapping Tools (GMT), which is a command
+line program that provides a wide range of tools for manipulating geospatial data and
+making publication-quality maps and figures. PyGMT integrates well with the
+[scientific Python ecosystem](https://scientific-python.org/), with [NumPy][] for its
+fundamental array data structure, [pandas][] for tabular data I/O and [Xarray][] for
+raster grids/images/cubes I/O.
+
+In addition to these core dependencies, PyGMT also relies on several optional packages to
+provide additional functionality for users.
+
+*This page was adapted from [GeoPandas's Ecosystem](https://geopandas.org/en/latest/community/ecosystem.html) page.*
+
+## PyGMT dependencies
+
+### Required dependencies
+
+#### NumPy
+
+[NumPy][] is the fundamental package for scientific computing in Python. It is a Python
```

Comment:
The NumPy description comes from https://numpy.org/doc/stable/user/whatisnumpy.html#whatisnumpy.